utils/formUtils.ts

```diff
@@ -7,6 +7,7 @@ import { Validator } from 'jsonschema'
 import { SplitSchema, Step, StepType } from '../types/interfaces'
 import RenderForm from '../src/Form/RenderForm'
 import RenderButtons from '../src/Form/RenderButtons'
+import { compose } from '@mui/system'
```

Comment:
Unused import?

Reply:
Not even sure how that got added, sometimes if you make a typo VS thinks it's an actual dependency and adds it!! :(